predictor/powerControl.yaml

```
@@ -0,0 +1,36 @@
+  PowerOffMaxNodesPerBatch: 10       
+  PowerOffBatchIntervalSeconds: 60    
+  ExcludeNodes:                    
+    - "craned2"
+  NodeBMCMapping:                 
+    craned175: "10.129.227.172"
+    craned1: "10.129.227.189"
+
+SSH:
+  User: "root"
+  Password: "hyq"
+  
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)